### DISCUSSION TITLE
Compact Kanban view

### DISCUSSION THREAD

**Discussion by jayaramkasi** [community/community#8926] [Projects and Issues]
Since the issue card is a bit stretched, if there are 8 or more issues, the layout gets stretched beyond 1 screen. Would be better to have a compact view as well which reduces the height and width of an issue to have a bird's eye view of the project.

**Comment by mustafaozhan** [comment]
Yes I think the Kanban view can be more compact also, tired of scrolling the UI needs to be improved, I currently zoom out in the browser for kanban view but it is not ideal since it effects all zoom level of github domain

**Reply by jamesj6150** [reply]
What they said
